apps/meteor/client/views/admin/moderation/helpers/ContextMessage.tsx

```diff
@@ -74,10 +76,17 @@ const ContextMessage = ({
 						{message.blocks && <UiKitSurface mid={message._id} blocks={message.blocks} appId rid={message.rid} />}
 						{message.attachments && <Attachments attachments={message.attachments} />}
 					</Message.Body>
+					<ReportReasonCollapsible>
+						<MessageReportInfo msgId={message._id} />
+					</ReportReasonCollapsible>
```

Comment:
`out-of-scope` This whole block could be inside the `<Message.Body>` above and implemented via `<UiKitSurface>`. We would get proper message content styling for free (e.g. user mentions).

Reply:
Sounds good to try. I will try this if I understood correctly; we move the 
`<ReportReasonCollapsible>...</ReportReasonCollapsible>` into the above `<Message.Body>` 
Sorry, but I am not able to understand the :
> implemented via <UiKitSurface>.

Reply:
As you can look some lines above, a `UiKitSurface` take an object describing interactive content that is supposed to be rendered in _surfaces_ like a message body, a modal, the banner etc. Is usually used by Rocket.Chat Apps as a GUI framework. You can see examples of it here: https://rocketchat.github.io/fuselage/uikit-playground/develop/

Reply:
Thank you! Oh, right, the Apps had this block of UI objects. Interesting; I will try to explore it.

Reply:
Actually, the `<MessageReportInfo />` renders a report reason and not the message itself, it takes the `msgId` and fetches all the report reasons. And the report reason is displayed as plain text. So it does not actually need the Message formatting.